external-api/src/v1/controllers/estore/eStore.controller.ts

```diff
@@ -1,48 +1,33 @@
-/* eslint-disable @typescript-eslint/unbound-method */
-/* eslint-disable @typescript-eslint/no-unsafe-call */
-/* eslint-disable @typescript-eslint/no-unsafe-member-access */
-/* eslint-disable @typescript-eslint/no-unsafe-assignment */
-/* eslint-disable @typescript-eslint/no-unsafe-argument */
-/* eslint-disable @typescript-eslint/no-explicit-any */
-/* eslint-disable @typescript-eslint/no-unsafe-return */
-/* eslint-disable no-param-reassign */
-/* eslint-disable @typescript-eslint/no-misused-promises */
-/* eslint-disable @typescript-eslint/no-floating-promises */
-/* eslint-disable import/no-extraneous-dependencies */
-
-import dotenv from 'dotenv';
 import { HttpStatusCode } from 'axios';
 import { ObjectId } from 'mongodb';
 import { Request, Response } from 'express';
-import { CronJob } from 'cron';
 import { getCollection } from '../../../database';
 import { Estore, SiteExistsResponse, EstoreErrorResponse } from '../../../interfaces';
 import { ESTORE_SITE_STATUS, ESTORE_CRON_STATUS } from '../../../constants';
 import { areValidUkefIds, objectIsEmpty } from '../../../helpers';
-import { eStoreTermStoreAndBuyerFolder, eStoreSiteCreationCron } from '../../../cron';
+import { eStoreTermStoreCreationJob, eStoreSiteCreationCron } from '../../../cron';
 import { createExporterSite, siteExists } from './eStoreApi';
 import { getNowAsEpoch } from '../../../helpers/date';
 
-dotenv.config();
-
-const { ESTORE_CRON_MANAGER_SCHEDULE, TZ } = process.env;
-
 export const create = async (req: Request, res: Response) => {
   try {
+    const { body } = req as {
+      body: object;
+    };
+
     // Ensure `req.body` is valid
-    if (objectIsEmpty(req.body)) {
+    if (objectIsEmpty(body)) {
       return res.status(HttpStatusCode.BadRequest).send({ status: HttpStatusCode.BadRequest, message: 'Invalid request' });
     }
 
     // Ensure new CRON job creation
     const cronJobLogs = await getCollection('cron-job-logs');
     const tfmDeals = await getCollection('tfm-deals');
 
-    const { dealId, siteId, facilityIdentifiers, supportingInformation, exporterName, buyerName, dealIdentifier, destinationMarket, riskMarket } = req.body;
-
-    let eStoreData = {} as Estore;
+    const { dealId, siteId, facilityIdentifiers, supportingInformation, exporterName, buyerName, dealIdentifier, destinationMarket, riskMarket } =
+      req.body as Estore;
 
-    eStoreData = {
+    const eStoreData: Estore = {
       dealId: new ObjectId(dealId),
```

Comment:
Replied further up but just re-iterating if we want to validate it's an objectID it's better to use ObjectId.isValid rather than making a new ObjectId